doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -1539,6 +1539,8 @@ In practice, you only need one the ``dataset_id`` or the ``persistentId``. The e
     print r.json()
     print r.status_code
 
+This API call might result in the following error: "Dataset store configuration does not allow provided storageIdentifier". One of the possible causes for this error is that out of band upload is not allowed for the storage that the provided identifier refers to. This error can be resolved by enabling the ``dataverse.files.\<id\>.upload-out-of-band`` JVM option. See :ref:`file-storage`.
```

Comment:
I'm confused - this call, which takes a file, can fail? It doesn't allow you to specify the storageidentifier does it?

Reply:
I have removed it to avoid the confusion. This error only happens when you uploaded a file on s3 (or some other storage) directly, and it is described in a separate section now (:doc:`/developers/s3-direct-upload-api` that uses exactly the same API call, but with storage identifier and without the file itself).